This package lists `argparse` as a separate dep, apparently from the days of
yore before it became part of the standard library. This messes up modern
`pip check` commands.

diff --git a/setup.py b/setup.py
index 696ab5fa27..851376c6fe 100644
--- a/setup.py
+++ b/setup.py
@@ -40,7 +40,6 @@ setup(name='ofxclient',
           ]
       },
       install_requires=[
-          "argparse",
           "keyring",
           "lxml",
           "ofxhome",
diff --git a/ofxclient.egg-info/requires.txt b/ofxclient.egg-info/requires.txt
index c6c7b4591a..4caef3fdd8 100644
--- a/ofxclient.egg-info/requires.txt
+++ b/ofxclient.egg-info/requires.txt
@@ -1,4 +1,3 @@
-argparse
 keyring
 lxml
 ofxhome
